samplers.go

```diff
@@ -112,6 +123,32 @@ func (s *Set) Flush() []DDMetric {
 	}}
 }
 
+func (s *Set) Export() (JSONMetric, error) {
+	val, err := s.hll.GobEncode()
+	if err != nil {
+		return JSONMetric{}, err
+	}
+	return JSONMetric{
+		MetricKey: MetricKey{
+			Name:       s.name,
+			Type:       "set",
+			JoinedTags: strings.Join(s.tags, ","),
+		},
+		Tags:  s.tags,
+		Value: val,
+	}, nil
+}
+
+func (s *Set) Combine(other []byte) error {
+	// you can only merge hyperloglogs with the same compression parameter
```

Comment:
yeah... now that I think about it, the merge error should probably be returned if it's not nil, since the decoded hyperloglog might have a different precision if it was from a weird client